dev/TeachingTip/TeachingTip.cpp

```diff
@@ -867,6 +952,29 @@ void TeachingTip::OnActionButtonClicked(const winrt::IInspectable&, const winrt:
 void TeachingTip::OnPopupOpened(const winrt::IInspectable&, const winrt::IInspectable&)
 {
     StartExpandToOpen();
+
+    if (auto peer = winrt::FrameworkElementAutomationPeer::FromElement(*this))
+    {
+        if (auto teachingTipPeer = peer.as<winrt::TeachingTipAutomationPeer>())
+        {
```

Comment:
combine these two statements with a try_as like you have elsewhere in this file?